### PR TITLE
Preparing release 1.5.0

### PR DESCRIPTION
**Pull request by jpkrohling**
Signed-off-by: Juraci Paixão Kröhling <juraci@kroehling.de>

## Which problem is this PR solving?
-  Release 1.5.0

## Short description of the changes
- User-visible changes from 1.4.1 to 1.5.0 added to the changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,26 @@
...
+#### Backend Changes
+
+- Add bounds to memory storage (#845) by Juraci Paixão Kröhling (@jpkrohling)
+- Add metric for debug traces (#796) by Won Jun Jang (@black-adder)
+- Change metrics naming scheme (#776) by Juraci Paixão Kröhling (@jpkrohling)
+- Remove ParentSpanID from domain model (#831) by Yuri Shkuro (@yurishkuro)
+- Add ability to adjust static sampling probabilities per operation (#827) by Won Jun Jang (@black-adder)
+- Support log-level flag on agent (#828) by Won Jun Jang (@black-adder)
+- Add healthcheck to standalone (#784) by Eundoo Song (@eundoosong)
+- Do not use KeyValue fields directly and use KeyValues as decorator only by Yuri Shkuro (@yurishkuro)
```
Comment (pavolloffay) [inline CHANGELOG.md]: missing PR reference

**Comment by pavolloffay** [comment]
I have created https://github.com/jaegertracing/jaeger/issues/848 to create guidelines

**Comment by jpkrohling** [comment]
PR updated to fix the reference to 810 in one of the commits

File: CHANGELOG.md (hunk 1 above)
Comment (jpkrohling) [reply CHANGELOG.md]: Updated

**Comment by pavolloffay** [comment]
I guess you don't want to update the links to fit with #848 so I will merge

**Comment by jpkrohling** [comment]
I would have no problems doing that, but I think this is a discussion we can have for the next release :)

**Pull request merged**